Codo-Baggins/covid_comparisons
# Testing/integration testing

## What's this PR do?
- Adds integration tests to `App`
- Runs test coverage

## Where should the reviewer start?
- `App.test.tsx`

## How should this be manually tested?
- Run `npm test` in the terminal
- Run `CI=true npm test -- --env=jsdom --coverage` in the terminal to see test coverage

## Any background context you want to provide?
- Will need to add integration test where user changes comparison selection and sees a new comparison page

## Questions:
- Any additional tests you want added?
